packages/components/app/styles/components/form/radio-card.scss

```
@@ -15,9 +15,18 @@
   }
 
   .hds-form-radio-card {
-    flex: 1;
     margin: calc(var(--token-form-radiocard-group-gap) / 2);
   }
+
+  // LAYOUT
+
+  .hds-form-radio-card--layout-fluid {
+    flex: 1;
+  }
+
+  .hds-form-radio-card--layout-fixed {
+    flex-basis: 100%;
+  }
```

Personally, when comes to `flex` I prefer explicit to implicit ;)

```suggestion
  .hds-form-radio-card--layout-fluid {
    flex: 1 0 0;
  }

  .hds-form-radio-card--layout-fixed {
    flex: 1 0 100%;
  }
```

Reason being, I always have to think of what the props are 😀